cfme/infrastructure/host.py

```diff
@@ -214,6 +217,12 @@ def delete(self, cancel=True):
         cfg_btn(btn_name, invokes_alert=True)
         sel.handle_alert(cancel=cancel)
 
+    def load_details(self, refresh=False):
+        """To be compatible with the Taggable and PolicyProfileAssignable mixins."""
+        navigate_to(self, 'Details')
```

Comment:
Nope, don't like it. Both `check_compliance` and `check_compliance_wrapper` should be removed not only from this PR but also from `cfme/common/vm.py`. It is 2 year old code that is not used anywhere.